python.d/rabbitmq.chart.py

```diff
@@ -72,6 +73,12 @@
         'lines': [
             ['proc_used', 'used', 'absolute']
         ]},
+    'erlang_run_queue': {
+        'options': [None, 'Erlang Run Queue', 'run_queue', 'overview',
```

Comment:
`run_queue` - average number of Erlang processes waiting to run. Not sure about 'run_queue' as units here.

Reply:
good point.  I was trying to not use 'processes' to avoid confusion, but that's probably better for consistency's sake.